composer/core/state.py

```diff
@@ -58,14 +58,8 @@ class State(Serializable):
 
     .. note::
 
-        * An instance of this class is automatically constructed by the :class:`~.Trainer` constructor. A user need
-          not instantiate this class.
-
-        * To support multi-GPU training, ``State.model`` may be wrapped in :class:`torch.nn.parallel.DistributedDataParallel`,
```

Comment:
These are somewhat out of date and a bit confusing, since it's the trainer that does this, not the state object.